src/util/util.cpp

```diff
@@ -72,6 +72,17 @@ void Util::update(QtCharts::QAbstractSeries* series, const QVector<double>& poin
     xySeries->replace(realPoints);
 }
 
+void Util::restartApplication()
```

Comment:
I mean did you use some example to create this, or did you create the solution yourself?

Reply:
Added.